tools/gluten-te/ubuntu/dockerfile-buildenv

```diff
@@ -81,10 +104,6 @@ ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en  
 ENV LC_ALL en_US.UTF-8
 
-# Install Spark 3.2.2
```

Comment:
why do we delete this?

Reply:
For speeding up debug CI dockerfiles. Compiling spark from source took too much time and spark from the maven repo worked just fine.

Reply:
several spark unite tests require compile spark(to get some resource file)